app/src/views/LoadingComponent.tsx

```diff
@@ -0,0 +1,45 @@
+import {CircularProgress, Typography, Stack} from '@mui/material';
+import {useState, useEffect} from 'react';
+
+export const LoadingComponent = () => {
+  const [text, showText] = useState<boolean>(false);
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      showText(true);
+    }, 5000);
+    return () => clearTimeout(timer);
+  }, []);
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        height: '100%',
+      }}
+    >
+      <Stack>
+        <div style={{display: 'flex', justifyContent: 'center'}}>
+          <CircularProgress value={25} />
+        </div>
```

Comment:
Can you get rid of this wrapping div and add the `justifyContent` property to the Stack parent component?

Reply:
Thanks Erik. I updated this.